magefiles/installation/install.go

```diff
@@ -149,6 +187,159 @@ func (i *InstallAppStudio) cloneInfraDeployments() (*git.Remote, error) {
 	return repo.CreateRemote(&config.RemoteConfig{Name: i.LocalForkName, URLs: []string{fmt.Sprintf("https://github.com/%s/infra-deployments.git", i.LocalGithubForkOrganization)}})
 }
 
+func MergePRInRemote(branch string, forkOrganization string, repoPath string) error {
+	if forkOrganization == "" {
+		forkOrganization = "redhat-appstudio"
```

Comment:
If we need to fallback to something (a.k.a. should we fall back, or fail here?), then I guess I would rather fall back to `redhat-appstudio-qe` org. WDYT?

Reply:
I have reworked this and this part is deleted